spec/database_helper.rb

```diff
@@ -0,0 +1,36 @@
+require "sequel"
+
+class DatabaseHelper
+  USER = ENV.fetch("DATABASE_USER")
```

Comment:
Sure, I'll try to setup a vagrant machine and use user/pass from there as the default in the tests.